src/Uno.UI/UI/Xaml/Controls/Slider/Slider.cs

```diff
@@ -352,7 +352,7 @@ private void OnSliderContainerPressed(object sender, PointerRoutedEventArgs e)
 
 			var newOffset = Orientation == Orientation.Horizontal ?
 				point.X / container.ActualWidth :
-				point.Y / container.ActualHeight;
+				1 - (point.Y / container.ActualHeight);
```

Comment:
So the easiest way to fix this, is to subtract the result from 1, inverting the inverted value. However, one could argue that the actual bug resides in the `ApplySlideToValue` method.

Reply:
Looks fine to me this way, I would prefer to keep the `ApplySlideToValue` logic the same for both horizontal and vertical orientations.